app/client/src/components/editorComponents/Debugger/helpers.tsx

```diff
@@ -120,6 +120,15 @@ export function getDependencyChain(
   return currentChain;
 }
 
+export const doesEntityHaveErrors = (
+  entityId: string,
+  debuggerErrors: Record<string, Message>,
+) => {
+  const ids = Object.keys(debuggerErrors);
+
+  return ids.some((e: string) => e.includes(entityId));
```

Comment:
nitpick: I guess this could be:
```suggestion
  return ids.some((e: string) => e.startsWith(entityId));
```